src/components/button/button.md.scss

```diff
@@ -470,7 +470,9 @@ $button-md-strong-font-weight:                  bold !default;
 // it's display none, and .md sets to display block.
 
 .button-effect {
-  @include position(0, null, null, 0);
+  // scss-lint:disable PropertySpelling
+  left: 0;
+  top: 0;
```

Comment:
First, know that this won't pass linting, as the properties are above an `@include`

Because of the new changes, this needs to be inside a `@include multi-dir() {` which goes at the end of the selector. When that is done, it can be merged. (Didn't merge before as I knew there will be changes with `multi-dir`)